starter-code/server.js

```diff
@@ -67,20 +83,29 @@ app.post('/articles', function(request, response) {
 });
 
 app.put('/articles/:id', function(request, response) {
-  // TODO: Write a SQL query to update an author record. Remember that our articles now have
+  // DONE TODO: Write a SQL query to update an author record. Remember that our articles now have
   // an author_id property, so we can reference it from the request.body.
```

Comment:
You need to retrieve the author by id so that you can use the table data to add the author name. This ensures that you use the right author name every-time rather than typing it out repeatedly. Then, you add the new article using the data that you got from the author table.